sklearn/feature_selection/_rfe.py

```diff
@@ -331,6 +333,15 @@ def _fit(self, X, y, step_score=None, **fit_params):
 
             estimator.fit(X[:, features], y, **fit_params)
 
+            # Compute step values on the previous selection iteration
+            # because 'estimator' must use features
+            # that have not been eliminated yet
```

Comment:
```suggestion
            # Compute step values on the previous selection iteration because
            # 'estimator' must use features that have not been eliminated yet
```